### DISCUSSION TITLE
Playbook,Alerts not working on 2.4

### DISCUSSION THREAD

**Discussion by GeorgeWere** [Security-Onion-Solutions/securityonion#10118] [2.4]
I have noticed i get an error trying to create a playbook. Also I cannot load any default/existing playbooks. The alerts also doesn't work. Not sure if its my installation that doesn't work or its a known issue. 

Otherwise the new version is off the hook, i love it even if its still on Beta

**Comment by dougburks** [comment]
Please try a fresh installation of 2.4 Beta 2 and see if it works better for you:
https://blog.securityonion.net/2023/04/security-onion-24-beta-2-release-now.html

**Reply by GeorgeWere** [reply]
Thanks Doug. Let me try it out
